src/script/conversation/ConversationRepository.ts

```diff
@@ -149,7 +148,6 @@ export class ConversationRepository {
   private readonly event_mapper: EventMapper;
   private readonly eventService: EventService;
   public leaveCall: (conversationId: QualifiedId, reason: LEAVE_CALL_REASON) => void;
-  private readonly receiving_queue: PromiseQueue;
```

Comment:
This receiving_queue didn't make sense for a while now (it's never paused anywhere, so there is no point queueing things here)